src/equations/equations.jl

```diff
@@ -133,38 +156,128 @@ See [`momentum`](@ref).
 
 varnames(::typeof(discharge), equations) = ("P",)
 
+"""
+    still_water_surface(q, equations::AbstractShallowWaterEquations)
+
+Return the still water surface ``\\eta_0`` (lake at rest)
+for a given set of `equations`.
+"""
+@inline function still_water_surface(q, equations::AbstractShallowWaterEquations)
+    return equations.eta0
+end
```

Comment:
We could consider adding
```julia
varnames(::typeof(still_water_surface), equations) = ("η_0",)
```
and similarly for `bathymetry` further down. If we do this, we should also add them to the `conversion_function`s in the unit tests.

Reply:
Since we don't use this right now, we may postpone it - or open an issue?